Testscripts/Windows/PERF-LKP.ps1

```diff
@@ -0,0 +1,248 @@
+                            -checkValues "PASS,FAIL,SKIP" -testName $CurrentTestData.testName
+                }
+            }
+            Remove-Item ".\tmp.txt"
+        }
+        return $resultPass
+
+    } catch {
+        Write-LogErr "Getting test summary results failed"
+        Remove-Item ".\tmp.txt"
+        $errorMessage =  $_.Exception.Message
+        $errorLine = $_.InvocationInfo.ScriptLineNumber
+        Write-LogErr "EXCEPTION : $errorMessage at line: $errorLine"
+        return $resultAborted
+    }
+}
+function Get-HorizontalFormat-TestResult ($currentTestResult, $currentTestData, $testName) {
+    try {
+        $isResultHead = $true
+        $metaDataTitle = "{0,-30}" -f "Jobs"
+
+        $LogPath = Join-Path $LogDir\$LKP_LOG_DIR $testName
+        if (-not (Test-Path -Path $LogPath)) {
+            Write-LogErr "EXCEPTION : $LKP_LOG_DIR\$testName doesn't exist"
+            return $resultAborted
+        }
+
+        $items = Get-ChildItem $LogPath
+        $items | ForEach-Object {
+            $resultPath = Join-Path $LogPath $_
+            $resultFile = Join-Path $resultPath "${testName}.json"
+            if (-not (Test-Path -Path $resultFile)) {
+                Write-LogErr "EXCEPTION : $resultFile doesn't exist"
+                return $resultPass
+            }
+            $jsonResult = (Get-Content $resultFile) | ConvertFrom-Json
+            $metaData = "{0,-30}" -f $_
+            $resultTitle, $result = Get-BenchmarkResult -testName $testName -jsonResult $jsonResult
+
+            if ($isResultHead) {
+                $isResultHead = $false
+                $item = "{0,-30} {1,30}" -f $metaDataTitle, $resultTitle
+                Add-Content -Value $item -Path "$LogDir\$LKP_RESULTS"
+                $CurrentTestResult.TestSummary += New-ResultSummary -testResult $resultTitle -metaData $metaDataTitle `
+                        -checkValues "PASS,FAIL,SKIP" -testName $CurrentTestData.testName
+            }
+
+            $item = "{0,-30} {1,30}" -f $metaData, $result
+            Add-Content -Value $item -Path "$LogDir\$LKP_RESULTS"
+            $CurrentTestResult.TestSummary += New-ResultSummary -testResult $result -metaData $metaData `
+                        -checkValues "PASS,FAIL,SKIP" -testName $CurrentTestData.testName
+       }
+
+       return $resultPass
+    } catch {
+        Write-LogErr "Getting test summary results failed"
+        $errorMessage =  $_.Exception.Message
+        $errorLine = $_.InvocationInfo.ScriptLineNumber
+        Write-LogErr "EXCEPTION : $errorMessage at line: $errorLine"
+        return $resultAborted
+    }
+}
+
+function Get-TestResult ($currentTestResult, $currentTestData) {
+    try {
+        Write-LogInfo "Uncompress $LKP_LOG_FILE..."
+        $Compress = .\Tools\7za.exe -y x "$LogDir\$LKP_LOG_FILE" -o"$LogDir"
+        if ( -not ($Compress -imatch "Everything is Ok")) {
+            Write-LogErr "EXCEPTION : Failed to uncompress $LKP_LOG_FILE"
```

Comment:
throw statement. 

Also if you are using Write-LogErr, I think this is associated to resultFailed. Please check with @LiliDeng

Reply:
No, not result failed, It's just the log colored red.